include/GafferUI/Gadget.h

```diff
@@ -88,13 +88,12 @@ class GAFFERUI_API Gadget : public Gaffer::GraphComponent
 
 		enum class Layer
 		{
-			None = -100,
-
-			Back = -2,
-			MidBack = -1,
-			Main = 0,
-			MidFront = 1,
-			Front = 2,
+			None = -1,
```

Comment:
Addressed in place in 6eff5b5.  I think I'm recalling correctly that when we talked it through, we decided not to worry much about headroom because we can adjust this while keeping source compatibility?  Or should I still bump it a bit?